docs/4.4/config-reference.md

```diff
@@ -180,6 +180,14 @@ auth_service:
     #    "node"  : sessions will be recorded on the node level  (the default)
     #    "proxy" : recording on the proxy level, see "recording proxy mode" section.
     #    "off"   : session recording is turned off
+    #
+    #    EXPERIMENTAL *-sync modes proxy and node send logs directly to S3 or other
```

Comment:
```suggestion
    # EXPERIMENTAL *-sync modes
    # Proxy and node send logs directly to S3 or other
```
This makes it clearer what the *-sync is referring to, there was no break in the sentence before.